tests/baselines/reference/for-of32.errors.txt

```diff
@@ -0,0 +1,7 @@
+tests/cases/conformance/es6/for-ofStatements/for-of32.ts(1,10): error TS7022: 'v' implicitly has type 'any' because it is does not have a type annotation and is referenced directly or indirectly in its own initializer.
+
+
+==== tests/cases/conformance/es6/for-ofStatements/for-of32.ts (1 errors) ====
+    for (var v of v) { }

```

Comment:
Also a great test. However, I realized that the semantics of this if you used a `let` binding could be different.

```TypeScript
let v = [1,2,3,4,5]
for (let v of v) {
    console.log(v);
}
```

If I try this out in Traceur, I get each element echoed to my console.